pyproject.toml

```diff
@@ -1,2 +1,108 @@
 [build-system]
 requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "hyp3lib"
+requires-python = ">=3.7"
+authors = [
+    {name="tools-bot", email="UAF-asf-apd@alaska.edu"},
+]
+description = "Common library for HyP3 plugins"
+license = {text = "BSD-3-Clause"}
+classifiers=[
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Software Development :: Libraries",
+]
+dependencies = [
+    "boto3",
+    "gdal",
+    "imageio",
+    "importlib_metadata",
+    "lxml",
+    "matplotlib",
+    "netCDF4",
+    "numpy<1.24",
+    "pillow",
+    "pyproj>=2",
+    "pyshp",
+    "requests",
+    "scipy",
+    "six",
+    "statsmodels",
+    "urllib3",
+]
+dynamic = ["version", "readme"]
```

Comment:
I don't see a reason to have a dynamic README -- simply doing:
```toml
[project]
...
readme = "README.md"
```
should be sufficient